### PR TITLE
#473

### PR DESCRIPTION
**Pull request by himmil**
Added colnames for samples if not defined to avoid unnecessary error in mae

### CODE REVIEW

File: R/getExperimentCrossAssociation.R [hunk 1]
```diff
@@ -308,6 +308,17 @@ setMethod("getExperimentCrossAssociation", signature = "SummarizedExperiment",
         } else {
             exp2 <- experiment2
         }
+        
+        # Add colnames if not defined
+        if (is.null(colnames(exp1))) {
+            colnames(exp1) <- paste("c", c(1:ncol(exp1)), sep = "")
```
Comment (TuomasBorman) [inline R/getExperimentCrossAssociation.R]: Use seq_len(10) instead of 1:10

**Comment by himmil** [comment]
Rownames don't need to be defined for functionality so I didn't edit them

**Comment by himmil** [comment]
@TuomasBorman ready to be merged

**Comment by TuomasBorman** [comment]
This leads to an error

```
data("GlobalPatterns")
tse <- GlobalPatterns
tse <- tse[1:10, ]

rownames(tse) <- NULL
getExperimentCrossAssociation(tse, tse)
```

**Comment by himmil** [comment]
It's fixed now @TuomasBorman

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**